.github/workflows/maven.yml

```diff
@@ -37,14 +37,50 @@ jobs:
         with:
           java-version: 17
           distribution: 'zulu'
-          cache: 'maven'
```

Comment:
I don't think we need the fully customised options of `Cache Maven packages` but could be wrong. I think this was fine.